src/ripple/app/paths/impl/DirectStep.cpp

```diff
@@ -283,7 +311,8 @@ void
 DirectStepI::setCacheLimiting (
```

Comment:
BTW, it's hard to see this in the github diff, but I think the paragraph comment above `setCacheLimiting()` is really useful.  Thanks.